modules/ROOT/pages/update-acm.adoc

```diff
@@ -303,6 +312,37 @@ Manual configuration:
 .. Click *Save*.
 . Verify that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
 
+Return to the main list of upgrade instructions to continue.
+
+[[post-20]]
+== Postinstallation Steps When Upgrading from a Version Earlier Than 2.0
+
+If you are upgrading from a package version earlier than 2.0, use either the automatic or manual configuration methods to update the permissions of the member and guest profiles. Both profiles require field-level read access to the *AssetId* field in the *AnypointApiInstances* object.
+
+Automatic configuration:
+
+. From the Anypoint API Community Manager control panel, open the Salesforce App Launcher and search for `API community manager diagnostics`.
+. In the *User Access* tab, in the section *Guest User*, click *Fix all*.
+. In the *User Access* tab, in the section *Member User*, click *Fix all*.
+. Verify that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
```

Comment:
Wait...you go through this whole task and then doublecheck it against some other doc?

Reply:
Yes. Customers have said the upgrade configurations are a pain point, and the ACM team is working on a comprehensive solution to automate all of this. The ACM team discussed whether to remove the comparison to the permissions on the community page, which increases the risk of errors, troubleshooting, or support cases, or keep the comparison, which makes the instructions more complex, and decided to keep it until the more comprehensive solution is released.